### DISCUSSION TITLE
[Suggestion] Make Wrecks a world event

### DISCUSSION THREAD

**Discussion by Udrakan** [FakeFishGames/Barotrauma#5713] [Ideas and suggestions]
![image](https://user-images.githubusercontent.com/59519434/117381589-b7864400-aedc-11eb-90b5-f3e2e062bb7c.png)
Similiar to how hunting grounds are on the map, players will see a wreck on the campaign map and decide to travel to it. This will make wrecks much more interisting due to rarity and being a "long term looting goal" for the overmap.

This might collide with Thalamus being the anti-lazy captain measure, but I think there might be a solution that would also solve the overall wreck loot abundancy: Spawn small very broken and looted+infested wrecks in all levels like now and spawn large (also possibly infested) wrecks with more loot on the campaign map that replace said wrecks when players travel there. Personally, I think there should only be wrecks from these world events, not in every level like it's now.

I'm imagining 2 blue wreck icons on the map: Small and Large wreck
Most wreck loot would be in the aphotic plateau/great sea both from gameplay and lore standpoint
Gameplay: Dangerous EVA combat and good loot later into campaign map
Lore: Further towards the Great Sea is harder for Coalition to recover wrecks and dangerous for looters to scavenge them.

Another problem this will solve is variety - players will experience wrecks much less often and because they will travel to them generally on purpose, its reasonable to believe they will go to them prepared (with intention). Thus it might be good to instead of focusing on massive variety of wrecks for replayability in Baro 1.0, focus on lesser amount of wrecks with memorable, even quite hard, monster encounters.

Discuss, modify, criticise

**Comment by Tsunder** [comment]
- I support more world-tier events. making the jumps/campaign level more interesting by making movement choices more interesting is important.
-- player choices (map movement) tangibly influencing level generation

- there'd need to be more separation of content tiers to find meaningful value in going one way or another
- -this is just baro in general being questionably balanced for the campaign scale of 20+ levels

- i don't think wrecks are passable as an anti-lazy captain thing. i think autopilot travels far enough away from wrecks to not get thalamus'd? 

- taking a card from a Certain Other greater scalar value C video game, I think it does campaign tier map really well by:
- - incomplete information about future movements
- - only one major event per map
- - it's also really short time-wise compared to baro campaigns lol

- ideas:
- - make world event tier stuff rare, like max 2 per biome
- - make it something special that isn't plain resources,
- - - perhaps a free NPC? some of those unique weapons (and remove them/greatly lower their % from the regular wreck pool)?

**Comment by Udrakan** [comment]
I agree and I think we can see this effect already with abandoned outposts. They generate quite a lot of excitement from what I've seen and people do significant detours just to get them on pub servers.
-they roughly equal to that once or twice per map "world event"
